_data/api/pro-sante-connect.md

```diff
@@ -0,0 +1,70 @@
+---
+title: API Pro Santé Connect
+tagline: Permettre l'authentification des professionnels de la santé
+external_site: https://esante.gouv.fr/
+is_open: -1 # -1 means API not open
+datapass_link: https://esante.gouv.fr/
+access_page:
+  - who:
+      - Un particulier
+    is_eligible: -1
+    description: |
+      Seules les administrations et les éditeurs de logiciels sont habilitées à utiliser l'API Pro Santé Connect ou intégrer le bouton Pro Santé Connect.
+
+      Si vous cherchez à contacter l'équipe Pro Santé Connect, vous pouvez [contacter l'Agence Numérique en Santé](https://esante.gouv.fr/assistance)
+
+      <Button href="/rechercher-api">Ou vous pouvez rechercher une autre API</Button>
+  - who:
+      - Une entreprise
+    is_eligible: 0
+    description: |
+      Seuls les éditeurs de logiciels proposant une solution web adressée aux professionnels de santé peuvent demander à se raccorder à PRO Santé Connect.
+      <NextSteps />
+      <Button href="https://franceconnect.gouv.fr/partenaires">Remplir une demande</Button>
+  - who:
+      - Une collectivité ou une administration
+    is_eligible: 1
+    description: |
+      Vous pouvez réaliser une demande d'accès à l'API. 
+      
```

Comment:
I advise to always have a CTA here

`<NextSteps />
<Button href="mailto:prosanteconnect.editeurs@esante.gouv.fr">Remplir une demande</Button>`